sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -303,19 +303,19 @@ export class ServiceBusManagementClient extends ServiceClient {
     deleteTopic(topicName: string, operationOptions?: OperationOptions): Promise<Response>;
     getNamespaceProperties(operationOptions?: OperationOptions): Promise<NamespacePropertiesResponse>;
     getQueue(queueName: string, operationOptions?: OperationOptions): Promise<QueueResponse>;
-    getQueueRuntimeInfo(queueName: string, operationOptions?: OperationOptions): Promise<QueueRuntimeInfoResponse>;
+    getQueueRuntimeProperties(queueName: string, operationOptions?: OperationOptions): Promise<QueueRuntimePropertiesResponse>;
```

Comment:
Changelog?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/10083/commits/8a253d5134da9ab8737e30f7d86ba3ceb8a40a62